db/seeds.rb

```diff
@@ -1,129 +1,214 @@
-      puts "Category #{i}: #{cat.name} created!"
-    end
+    Category.create(
+    name: "Dodgeball",
+    description: "Stupid Sports"
+    )
+
+    Category.create(
+    name: "Tazerball",
+    description: "Horrible Music"
+    )
+
+    Category.create(
+    name: "Robot Fighting Championships",
+    description: "Crazy Events"
+    )
+
+    Category.create(
+    name: "RC Truck Racing",
+    description: "Stupid Sports"
+    )
+
+    Category.create(
+    name: "3rd Person Skiing",
+    description: "Horrible Music"
+    )
+
+    Category.create(
+    name: "Chess Boxing",
+    description: "Crazy Events"
+    )
+
+    Category.create(
+    name: "Freestyle Canoeing",
+    description: "Stupid Sports"
+    )
+
+    Category.create(
+    name: "Combat Juggling",
+    description: "Horrible Music"
+    )
+
+    Category.create(
+    name: "Lawnmower Ice Racing",
+    description: "Crazy Events"
+    )
+
+    Category.create(
+    name: "Head Pong",
+    description: "Stupid Sports"
+    )
   end
 
   def generate_users
-      100.times do |i|
-        user = User.create!(
-          username: Faker::Name.name,
-          email: Faker::Internet.email,
-          first_name: Faker::Name.name,
-          last_name: Faker::Name.name,
-          password: "password",
-        )
+    99.times do |i|
+      user = User.create!(
+      username: Faker::Internet.user_name,
+      email: Faker::Internet.email,
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      password: "password",
+      )
+      if i < 19
         user.roles << Role.find_by(name: "store_admin")
-        puts "User #{i}: #{user.username} - #{user.email} created!"
+      else
+        user.roles << Role.find_by(name: "registered_user")
+      end
+      puts "User #{i}: #{user.username} - #{user.email} created!"
     end
   end
 
   def generate_events
-      500.times do |i|
-        vendors  = Role.find_by(name: "store_admin").users
-        vendor = vendors.sample
-        event = vendor.events.create!(
-          name: "#{Faker::Commerce.product_name}_#{i}" ,
-          description: Faker::Lorem.paragraph,
-          image_url: "http://loremflickr.com/320/240/sports?random=#{i}",
-          price: Faker::Commerce.price,
-          status: 0,
-          venue: Faker::Address.city,
-          event_date: Faker::Date.forward((1..300).to_a.sample),
-          category_id: Random.new.rand(1..28)
-        )
-        puts "Event #{i}: #{event.name} created!"
-    end
-  end
-
-  def generate_orders
-    100.times do |i|
+    500.times do |i|
       vendors  = Role.find_by(name: "store_admin").users
       vendor = vendors.sample
-      customer = User.find(Random.new.rand(1..100))
-      order = vendor.orders.create(customer_id: customer.id)
-      add_events(order)
-      puts "Order #{i}: Order for #{vendor.username} created!"
+      event = vendor.events.create!(
+      name: "#{Faker::Commerce.product_name}_#{i}" ,
```

Comment:
Indent the first parameter one step more than the previous line.<br>Space found before comma.